### ISSUE TITLE
Discord links in `projects/GETTING_STARTED_DWP.md` not working

### ISSUE THREAD

**Issue by Fittiboy** [TBD54566975/collaboration#35]
The links are direct links to channels, which only work for users who are already on the server!
Without a server invite, these links aren't usable! Same goes for [the website](https://developer.tbd.website/projects/web5/).

**Comment by rektide** [comment]
None of the discord links work. anywhere, from what I've seen.

**Comment by Fittiboy** [comment]
Yeah I'm assuming that every Discord link throughout the project has the same problem! So someone should go through them and replace the links! You can actually create invites to specific channels.
Just have to make sure they're unlimited-use invites without a timeout!

**Comment by profishional** [comment]
Only the links from the main landing pages work: https://discord.gg/MASAPKPYJM. Otherwise, these are channel links.

**Comment by ALRubinger** [comment]
Thanks, all, for the report!

I've combed through the links and found the following current state:

* Footer in developer.tbd.website: https://discord.gg/tbd
* Footer in tbd.website: https://discord.gg/MASAPKPYJM - incorrect as deployed, opened issue to address, corrected in CMS by @mistermoe 

For the Markdown links in this repo as noted, I'll address those here with commits.

**Comment by ALRubinger** [comment]
Resolved through linked commits.
